flutter/ios/Classes/SwiftSentryFlutterPlugin.swift

```diff
@@ -121,6 +124,13 @@ public class SwiftSentryFlutterPlugin: NSObject, FlutterPlugin {
         result("")
     }
 
+    private func closeNativeSdk(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
+        // swiftlint:disable:next todo
+        // TODO: Call SentrySDK when available
```

Comment:
We can merge this and keep sentry-cocoa as no-op for now.

Reply:
@philipphofmann can u tell us how to swap the `enabled` flag? could be a workaround for now

is it possible to mutate this https://github.com/getsentry/sentry-cocoa/blob/master/Sources/Sentry/SentryOptions.m#L22 after SDK is init?

Reply:
Yes, you can swap the enabled flag. The client is then just dropping everything. See https://github.com/getsentry/sentry-cocoa/blob/789d52b29386cb0267be6b9696ce29df038f40dc/Sources/Sentry/SentryClient.m#L448-L451.

Reply:
@philipphofmann yep, we'd like to know `how` to swap it, I only see a getter but not a setter across SentrySDK/Hub and Client

Reply:
We made the [SentrySDK currentHub] private in 7.x.x. For Hybrid SDKs, it is fine to create a category to make this method visible, like https://github.com/getsentry/sentry-cocoa/blob/master/Sources/Sentry/include/SentrySDK+Private.h.  Then you should be able to do something like `SentrySDK.currentHub().getClient()?.options.enabled = false`. It's not nice, but it should work.

Reply:
@philipphofmann we'd do the 1st iteration without v7, so we could do this directly right now, thanks.

`SentrySDK.currentHub().getClient()?.options.enabled = false`

Once v7 is GA, we'll probably have a proper SentrySDK.close() method and could get rid of it altogether.